project.clj

```diff
@@ -8,7 +8,7 @@
 
   :min-lein-version "2.7.1"
```

Comment:
Do we need to bump this to 2.9.0 as thats where nrepl/nrepl was folded into lein?

Reply:
Sure. If we don't we should probably remove the line about it in the CHANGELOG.

Reply:
Bumped to 2.9.0.